styles/tools/functions_test.scss

```
@@ -1,9 +1,26 @@
 @import "node_modules/sass-true/sass/true";
 @import "./functions";
 @import "./../settings/colors";
+@import "./../settings/font-families";
+@import "./../settings/font-sizes";
 
 @include describe("color function") {
   @include it("returns hex value from colors scss map") {
     @include assert-equal(color(blue-100), #00abc5);
   }
 }
+
+@include describe("font-size function") {
+  @include it("returns rem value from font-size scss map") {
+    @include assert-equal(font-size(sm),
+    1.063rem);
```

I would put all of this on one line, just for readability.

```suggestion
    @include assert-equal(font-size(sm), 1.063rem);
```